homework_2/turtle-stretch.js

```diff
@@ -0,0 +1,278 @@
+/* [Homework] Turtle Graphics - 2 */
+
+// Require the file system module
+const fs = require('fs');
+
+// Create the turtle class
+class Turtle {
+  // Initialize the constructor to the default values if provided
+  constructor(x, y) {
```

Comment:
If you want to instantiate `Turtle` the way you did in line 220, it's advised that you do `constructor(x = 0, y = 0)` or do `let turtle = new Turtle(0, 0)` in line 220 because the constructor requires two parameters.